apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -396,7 +396,7 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
             { !isLoadingForTheFirstTime? (
                 <>
                     { renderTenantedMyAccountLink() }
-                    { renderRemoteFetchStatus() }
+                    { /* renderRemoteFetchStatus() */ }
```

Comment:
Shall we link this issue here.
https://github.com/wso2/product-is/issues/11320#issuecomment-907163540

IMO, better if we can disable the feature from the feature config via default.json (carbon-identity-framework).